package.json

```diff
@@ -13,13 +13,13 @@
     "package-term": "yarn workspace @gravitational/teleterm package",
     "storybook": "start-storybook -p 9002 -c web/.storybook -s web/.storybook/public",
     "storybook-smoke-test": "yarn storybook --ci --smoke-test",
-    "test": "jest",
+    "test": "yarn build-wasm && jest",
```

Comment:
Adding a new build tool requirement is something that should be called out loudly. It seems rather hidden in this PR titled "Fix yarn test" where the PR description does not mention it either. I'm not sure of the relationship between the rust `wasm-pack` package/module and the yarn `build-wasm` command so I think that should be documented somewhere somehow too.

Reply:
These were removed in favor of running them as a separate step in the CI pipeline https://github.com/gravitational/teleport/pull/35401/commits/9a3579c00fd89c4253d93d545ea06d1bc75820cd

This is because we don't necessarily want to ensure wasm is built before _every_ run of `yarn test`, e.g. sometimes we just want to do `yarn test <specific test which may not even rely on wasm modules>